### ISSUE TITLE
Upgrade to v2.0.0 requires different webpack configuration

### ISSUE THREAD

**Issue by mircowidmer** [francisbesset/font-awesome-sass-loader#6]
After upgrading to v2.0.0 I changed my webpack configuration to:

```
// the url-loader uses DataUrls.
// the file-loader emits files.
{
  test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
  options: {
    limit: 10000,
    mimetype: 'application/font-woff'
  }
},
{
  test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
  use: [
    { loader: 'file-loader' }
  ]
}
```
as provided in the README. Unfortunately that resulted in the error 
```
Error: options/query provided without loader (use loader + options) in {
  "test": {},
  "options": {
    "limit": 10000,
    "mimetype": "application/font-woff"
  }
}
```
while running with `npm run serve`.

The following configuration seems to work now:

```
// the url-loader uses DataUrls.
// the file-loader emits files.
{
  test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
  options: {
    limit: 10000,
    mimetype: 'application/font-woff'
  },
  loader: 'url-loader'
},

{
  test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
  loader: 'file-loader'
}
```

**Comment by francisbesset** [comment]
Yes I'm sorry, I went too fast yesterday.

The good configuration is:
```javascript
module.exports = {
  module: {
    rules: [
      // the url-loader uses DataUrls.
      // the file-loader emits files.
      {
        test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
        use: [
          {
            loader: 'url-loader',
            options: {
              limit: 10000,
              mimetype: 'application/font-woff'
            }
          }
        ]
      },
      {
        test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
        use: [
          { loader: 'file-loader' }
        ]
      }
    ]
  }
};
```

**Comment by francisbesset** [comment]
README fixed in v2.0.1 :)
